src/sss_client/pam_message.h

```diff
@@ -66,6 +66,7 @@ struct pam_items {
     char *first_factor;
     char *passkey_key;
     char *passkey_prompt_pin;
+    uint32_t attempts;
```

Comment:
This is pretty messy structure :-/

Would it make sense to add 'passkey_' or 'passkey_pin_' prefix to 'attempts'?
Or is this going to be reused outside of passkey context?